templates/index.html

```diff
@@ -25,111 +25,147 @@
+            <p class="p-matrix__desc">A pure-container hypervisor. Replace legacy app VMs with containers for speed and density</p>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <a href="https://landscape.canonical.com/static/doc/user-guide/">
+            <img class="p-matrix__img" src="https://assets.ubuntu.com/v1/4964d639-landscape-logo.svg" alt="Icon">
+          </a>
+          <div class="p-matrix__content">
+            <h4 class="p-matrix__title"><a class="p-link" href="https://landscape.canonical.com/static/doc/user-guide/">Landscape&nbsp;</a></h4>
+            <p class="p-matrix__desc">Systems management for Ubuntu - updates, package management, repositories, security</p>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <a href="https://docs.conjure-up.io/2.4.0/en/">
+            <img class="p-matrix__img" src="https://assets.ubuntu.com/v1/c5cb0f8e-picto-ubuntu.svg" alt="Icon">
+          </a>
+          <div class="p-matrix__content">
+            <h4 class="p-matrix__title"><a class="p-link" href="https://docs.conjure-up.io/2.4.0/en/">Conjure-up&nbsp;</a></h4>
+            <p class="p-matrix__desc">Use conjure-up to get you a fully installed and usable stack</p>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <a href="https://wiki.ubuntu.com/Mir">
+            <img class="p-matrix__img" src="https://assets.ubuntu.com/v1/c5cb0f8e-picto-ubuntu.svg" alt="Icon">
+          </a>
+          <div class="p-matrix__content">
+            <h4 class="p-matrix__title"><a class="p-link" href="https://wiki.ubuntu.com/Mir">MIR&nbsp;</a></h4>
+            <p class="p-matrix__desc">Ultra-fast and light Wayland compositor for secure device display management</p>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <a href="https://github.com/CanonicalLtd/multipass">
+            <img class="p-matrix__img" src="https://assets.ubuntu.com/v1/c5cb0f8e-picto-ubuntu.svg" alt="Icon">
+          </a>
+          <div class="p-matrix__content">
+            <h4 class="p-matrix__title"><a class="p-link" href="https://github.com/CanonicalLtd/multipass">Multipass&nbsp;</a></h4>
+            <p class="p-matrix__desc">Ubuntu virtual machines on demand for Linux, MacOS and Windows</p>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <a href="https://help.ubuntu.com/community/CloudInit">
+            <img class="p-matrix__img" src="https://assets.ubuntu.com/v1/c5cb0f8e-picto-ubuntu.svg" alt="Icon">
+          </a>
+          <div class="p-matrix__content">
+            <h4 class="p-matrix__title"><a class="p-link" href="https://help.ubuntu.com/community/CloudInit">Cloud-init&nbsp;</a></h4>
+            <p class="p-matrix__desc">Apply user data to your instances automatically</p>
+          </div>
+        </li>
+      </ul>
+    </div>
+    <div class="row">
```

Comment:
I think we can split this out into a separate strip, and remove the top padding to keep the spacing consistent with the top and bottom i.e. `<div class="p-strip is-bordered u-no-margin--top">